app/services/create_link.rb

```diff
@@ -0,0 +1,25 @@
+require 'uri/http'
```

Comment:
This service class doesn't appear to use `uri/http`, and so I think this require statement can be removed.